### DISCUSSION TITLE
Add message in AR screen

### DISCUSSION THREAD

**Discussion by BelenSeo** [google/model-viewer#4180] [Q&A]
Hello! I need to add user  indications in de AR screen of <model-viewer>, and also change the hand animation inside the AR screen. How can this be done? I am working in react

**Comment by elalish** [comment]
It's only possible to customize the WebXR AR mode (Android-only). See this example, which adds a hand animation and such using HTML and CSS: https://modelviewer.dev/examples/augmentedreality/#webXR
